src/main/java/io/vertx/ext/mongo/IndexOptionDefaults.java

```diff
@@ -1,68 +0,0 @@
-package io.vertx.ext.mongo;
-
-import io.vertx.codegen.annotations.DataObject;
-import io.vertx.core.json.JsonObject;
-
-import java.util.Objects;
-
-/**
- * The default options for a collection to apply on the creation of indexes.
- */
-@DataObject(generateConverter = true)
-public class IndexOptionDefaults {
```

Comment:
This is a breaking change.

Reply:
Not really. This class was previously used in a commit and its usage was removed as of your feedback https://github.com/vert-x3/vertx-mongo-client/pull/266#discussion_r760285425

So it's an orphaned data object, which is not used in the whole project. I'd not consider this a breaking change.

Reply:
Ok, if it's a leftover you can remove it and add a note to the wiki page.